yt_dlp/extractor/news9.py

```diff
@@ -0,0 +1,70 @@
+from .common import InfoExtractor
+from .brightcove import BrightcoveNewIE
+
+from ..utils import ExtractorError, traverse_obj
+
+
+class News9IE(InfoExtractor):
```

Comment:
Should we name the extractor `NineNewsIE` instead? That's also what [Wikipedia](https://en.wikipedia.org/wiki/Nine_News) calls the outlet.

Reply:
Thanks for the suggestion. Makes more sense than reversing the name.